config/vm_repo.go

```diff
@@ -63,3 +66,37 @@ func (r vMRepo) ClearCurrent() error {
 	}
 	return nil
 }
+
+func (r vMRepo) FindCurrentIP() (string, bool, error) {
+	deploymentState, err := r.deploymentStateService.Load()
+	if err != nil {
+		return "", false, bosherr.WrapError(err, "Loading existing config")
+	}
+
+	currentIP := deploymentState.CurrentIP
+	if currentIP != "" {
+		parsedIP := net.ParseIP(currentIP)
+		if parsedIP == nil {
+			return "", false, bosherr.Errorf("%v is not a valid IP address", currentIP)
+		} else {
+			return parsedIP.String(), true, nil
+		}
+	}
+
+	return "", false, nil
+}
+
+func (r vMRepo) UpdateCurrentIP(ip string) error {
```

Comment:
We don't see anywhere where this is called? Is this an oversight on our end, or is this not used?